src/Creator.tsx

```diff
@@ -145,6 +148,35 @@ export const Creator = ({ NFToupon_Key }: Props) => {
   };
 
   const sendDetails = async () => {
+    if (textAreaValue === '') {
```

Comment:
what if i enter spaces this condition will be false in that case which is wrong... we need to prevent that as well..